src/style.css

```diff
@@ -0,0 +1,65 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  justify-content: center;
+
+  height: 300px;
+  min-width: 300px;
+  width: 33.33%;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+@media (min-width: 600px) {
+  .block-3,
+  .block-5,
+  .block-6 {
+    order: 1;
+  }
+}
+
+@media (min-width: 900px) {
+  .block-5 {
+    order: 4;
+  }
+  .block-4 {
+    order: 5;
+  }
+}
```

Comment:
due to all the other elements have order either 0 or 1, you could have put 2 and 3 here